person_capture/gui_app.py

```
@@ -5429,6 +5453,80 @@ def save_hit(
+                                    pad_y_top=0.10 * hfh,
+                                    pad_y_bottom=0.16 * hfh,
+                                    bounds_xyxy=(repair_bx1, repair_by1, repair_bx2, repair_by2),
+                                ) or hf
+                                cur_crop = (float(cx1), float(cy1), float(cx2), float(cy2))
+                                cur_w = max(1.0, float(cx2 - cx1))
+                                cur_h = max(1.0, float(cy2 - cy1))
+                                cur_face_h_frac = hfh / cur_h
+                                try:
+                                    rrw, rrh = parse_ratio(ratio_str)
+                                    cur_aspect = float(rrw) / max(1e-6, float(rrh))
+                                except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)